xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -229,7 +229,11 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
         <input type="hidden" name="${xCommentClass}_replyto" value="$!replyTo"/>
         <div class="commentcontainer">
           <div class="commenteditor">
-              <!-- loaded by javascript (see comments.js) -->
+            <!-- loaded by javascript (see comments.js) -->
+            <!-- stores the submitted comment in case of error during the submission (eg, invalid captcha). -->
+            #if ($captchaAnswerWrong)
+              <div class="hidden" id="submittedcomment">$!comment</div>
```

Comment:
Ok so now it could be really an input hidden field no?

Reply:
I mean you always want to send it back in this case.

Reply:
Replaced by a hidden field.